.github/workflows/docker-build-static.yml

```diff
@@ -91,6 +91,8 @@ jobs:
         run: ./app/build.sh
       - name: check path
         run: ls -al app/
+      - name: overwrite index.html with index_local.html
```

Comment:
Shouldn't that be part of the Dockerfile?

Reply:
This can both ways, it just when doing via CI it affect both container directly instead modifying two Dockerfiles. 🤔

Reply:
I'd rather keep it in the Dockerfiles: this is part of building the image, not part of checking it.